src/edu/ucsb/nceas/metacat/healthchecks/StartupRequirementsListener.java

```diff
@@ -85,24 +96,28 @@ public void contextDestroyed(ServletContextEvent sce) {
      *
      * (protected to allow test access)
      * @param sce the ServletContextEvent passed by the container on startup
-     * @return Path object pointing to 'metacat-site.properties'. Will never be null
+     * @return a Properties object initialized from the 'metacat.properties' file
      * @throws RuntimeException if any unrecoverable problems are found that should cause startup
      *                          to be aborted
      */
-    protected Path validateDefaultProperties(ServletContextEvent sce) throws RuntimeException {
+    protected Properties validateDefaultProperties(@NotNull ServletContextEvent sce)
+        throws RuntimeException {
 
         Path defaultPropsFilePath =
             Paths.get(sce.getServletContext().getRealPath("/WEB-INF"), "metacat.properties");
-        Properties metacatProperties = new Properties();
-        Path sitePropsFilePath = null;
+        Properties defaultProperties = new Properties();
         try {
-            metacatProperties.load(Files.newBufferedReader(defaultPropsFilePath));
-            metacatProperties.store(Files.newBufferedWriter(defaultPropsFilePath), "");
-
-            sitePropsFilePath = Paths.get(
-                metacatProperties.getProperty(PropertyService.SITE_PROPERTIES_DIR_PATH_KEY),
-                PropertyService.SITE_PROPERTIES_FILENAME);
-
+            // can read?
+            defaultProperties.load(Files.newBufferedReader(defaultPropsFilePath));
+            // can write? Use isWriteable() so we don't mess up props file formatting for Jing :-)
```

Comment:
Thanks. But the metacat.properties file is still changed  :).  I believe it happens when we write some properties (such as status) into the file.

Reply:
agreed - I just thought I would be better not to have it happen in yet another place :-)